projects/module-3/stretch.md

```diff
@@ -246,151 +246,64 @@ Evals are
 * 1:1 sessions with you and an instructor from the Frontend Team
 * 15 minutes
 * Two interview questions - one technical (about your stretch tech) and one behavioral (you get to choose the "theme", but not the specific question)
-* You will be expected to send your interviewer your resume prior to the interview via slack
+* You will be expected to send your interviewer your resume (updated with the stretch project) prior to the interview via slack
 
 ## Rubric
 
 There will be no written feedback for this project, but each and every one of you is capable of self-assessing whether or not your project is proficient. Mentors and Rocks are also a valuable resource here. **A Proficient project would include the following:**  
-### Professionalism/Workflow:
-- [ ] Project is deployed
-- [ ] README concisely communicates the team's individual and joint learning goals, the evolution of the project, and team member reflections while using good formatting to enhance readability
-- [ ] README links to all user GitHub profiles and any applicable repos & deployed sites
-- [ ] Setup instructions for any and all repos are thorough and verbose enough that even non-technical people (like recruiters) could follow them
-- [ ] Git commits are atomic, with concise and precise descriptions of the change made
-- [ ] PRs have full, consistent descriptions
-- [ ] Team members conduct consistent code reviews of PRs
-- [ ] Evolution of the project (decisions made, etc) is documented in the git history and PRs
-- [ ] When the project is run locally, the terminal shows no errors and no warnings  
+### Minimum Collaboration and Professionalism Expectations
+
```

Comment:
I haven't seen minimum professionalism expectations laid out in the On Track vs Wow systems like this. I wonder if we do want students picking and choosing which bullet points to complete to illustrate competency or if we do want _all_ minimum requirements met for professionalism and collaboration. The On Track vs Wow system implies that there are many ways to prove competency, but our other rubrics (such as the M2 rubrics: https://frontend.turing.edu/projects/module-2/fitlit-part-one-agile.html) imply that the professionalism and collaboration items are all required. 

This is just food for thought. 

My vote is to keep this rubric like it is and test it out. If students are still performing well or better, then maybe it would be good to have all the rubrics look like this. If not, we can change it to requirements.